pyproject.toml

```diff
@@ -75,8 +75,14 @@ types-requests = "^2.31.0"
 types-pyOpenSSL = "^24.1.0"
 types-setuptools = "^75.6.0"
 
+[tool.mypy]
+ignore_missing_imports = true
+
 [tool.ruff.lint]
 ignore = ["F401"]
+# Ruff defaults (F + some E), as well as I (isort) and W.
+# Maybe consider adding some subset of "D" here as well
+select = ["E4", "E7", "E9", "F", "I", "W"]
```

Comment:
not 100% sure if `select` overrides defaults or is additive, but seemed best to just be explicit here.

Reply:
"Explicit is better than implicit." :)